### PR TITLE
Add /fees/nft endpoint

### LINKED ISSUE #621: API: Fee Amounts for NFTs
* Return the fee amounts for NFTs, both for Hedera and the corresponding wrapped networks.
    * For wrapped NFTs, the payment token + amount will need to be fetched from the Router Smart Contract 

* Might be added either in `min-amounts` or in another endpoint/

### PR DESCRIPTION
**Pull request by vectorman1**
Signed-off-by: Boris Lachev <boris.lachev@icloud.com>

**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #621 

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [x] Tests updated

### CODE REVIEW

File: app/router/fees/fees.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package fees
```
Comment (svetlio8) [inline app/router/fees/fees.go:1]: Missing license

**Review by svetlio8 — APPROVED** [review]

File: app/services/pricing/service.go [hunk 2]
```diff
@@ -348,5 +466,10 @@ func initialize(bridgeConfig *config.Bridge, assetsService service.Assets, mirro
 	if err != nil {
 		panic(fmt.Sprintf("Failed to initially fetch USD prices. Error: [%s]", err.Error()))
 	}
+
+	err = instance.FetchAndUpdateNftFeesForApi()
+	if err != nil {
+		panic(fmt.Sprintf("Failed to initially fetch portal/burn fees for NFT assets. Error: [%s]", err.Error()))
+	}
```
Comment (failfmi) [inline app/services/pricing/service.go:473]: You call this in `instance.FetchAndUpdateUsdPrices()`, is there any point this to still be called here?

File: app/services/pricing/service_test.go [hunk 3]
```diff
@@ -22,6 +22,12 @@ import (
 	"sync"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+
+	client2 "github.com/limechain/hedera-eth-bridge-validator/app/domain/client"
```
Comment (failfmi) [inline app/services/pricing/service_test.go:29]: Can we rename this to something more meaningful?

File: app/services/pricing/service.go [hunk 4]
```diff
@@ -103,6 +106,121 @@ func (s *Service) FetchAndUpdateUsdPrices() error {
 	return nil
 }
 
+func (s *Service) FetchAndUpdateNftFeesForApi() error {
```
Comment (failfmi) [inline app/services/pricing/service.go:109]: nit: we can lower the visibility of this function

File: app/domain/service/pricing.go [hunk 5]
```diff
@@ -29,4 +29,9 @@ type Pricing interface {
 	GetMinAmountsForAPI() map[uint64]map[string]string
 	// GetHederaNftFee returns the nft fee for Hedera NFTs based on token id
 	GetHederaNftFee(token string) (int64, bool)
+
+	// FetchAndUpdateNftFeesForApi fetches the fees for porting/burning NFTs
+	fetchAndUpdateNftFeesForApi() error
```
Comment (failfmi) [inline app/domain/service/pricing.go:34]: I guess this is no longer needed.

**Review by failfmi — CHANGES_REQUESTED** [review]

**Review by failfmi — APPROVED** [review]

**Pull request merged**